### DISCUSSION TITLE
Start Menu No Longer Displays as Windows 10

### DISCUSSION THREAD

**Discussion by ViktorSwanson** [valinet/ExplorerPatcher#3601] [General]
I first downloaded ExplorerPatcher for the sole purpose of enabling the windows 10 start menu. After installing 5 days ago I enabled "windows 10 start menu" and organized the start menu exactly how I liked it. Today, after updating windows and restarting a few times, I lost the option in  the ep Properties to even select the windows 10 start menu. 

I exported my settings and reinstalled ep, but the issue still persists. 

![image](https://github.com/user-attachments/assets/b2f3ef1a-17f1-4eea-a190-90d82189044c)

Any help is appreciated, the windows 11 start menu is unbelievably ugly and I was starting to think I could get used to the windows 11 after retrofitting the start menu to windows 10.

**Comment by pyrates999** [comment]
Install the latest pre-release: https://github.com/valinet/ExplorerPatcher/releases

**Reply by ViktorSwanson** [reply]
I tried to download this pre-release and it set off every antivirus alarm on my PC, and I am cautious, why wouldn't the latest release work?

**Reply by pyrates999** [reply]
read the release notes for it first before you download it.

**Reply by milestprower92** [reply]
> I tried to download this pre-release and it set off every antivirus alarm on my PC, and I am cautious, why wouldn't the latest release work?

you will need to put ep_setup.exe on the exclusion list of your antivirus software
(the reason why antivirus software specially Defender flagged EP is because Microsoft doesn't want users to use the WIndows 10 shell on Windows 11)

**Reply by ViktorSwanson** [reply]
There we go, my beloved windows 10 start menu is back! Much appreciated fellas!

**Reply by pyrates999** [reply]
Please close this.
